src/style/global.scss

```diff
@@ -51,10 +51,6 @@ html {
   }
 }
 
-.iui-anchor {
```

Comment:
So if users doesn't import `anchor.css` and they previously used `iui-anchor` class from global css, their anchors will break. But maybe that's ok with pre major release of css...

Reply:
There would be no change for those who use `all.css` though, I think that's a plus.